bot/cogs/doc.py

```diff
@@ -243,25 +305,39 @@ def __init__(self, bot: commands.Bot):
         if len(description) > 1000:
             shortened = description[:1000]
             last_paragraph_end = shortened.rfind('\n\n')
-            description = description[:last_paragraph_end] + f"... [read more]({permalink})"
+            description = description[:last_paragraph_end]
```

Comment:
If a paragraph ending is never found, the content is never shortened. This could possibly result in embed content being above 2000 characters.

Reply:
resolved in 4795da86d0fef72ac677ae0a8f9e988da1923e17